deploy/kubernetes/README.md

```diff
@@ -4,7 +4,17 @@ cAdvisor uses [Kustomize](https://github.com/kubernetes-sigs/kustomize) to manag
 
 ## Usage
 
-From the cadvisor/ directory, to generate the base daemonset:
+Pick a [cAdvisor release](https://github.com/google/cadvisor/releases)
+```
+VERSION=v0.42.0
+```
+
+From the `cadvisor/` directory, to update the image version([reference](https://github.com/kubernetes-sigs/kustomize/blob/master/examples/image.md)):
```

Comment:
Now it looks better but I think that you don't need "From the `cadvisor/` directory..." cause it's obvious that we're running commands from the repository directory.
Could you change also line 17?

Reply:
Updated.